src/layouts/AppLayout.vue

```diff
@@ -30,9 +49,47 @@ const pageTitle = computed(() => route.meta.title);
     padding: 16px;
     margin: 0 auto;
     max-width: $desktop-upper-breakpoint;
-    .title {
-      font-size: $h2-font-size;
+    .header {
+      width: 100%;
       margin-bottom: 16px;
+      display: flex;
+      justify-content: space-between;
+
+      .title {
+        font-size: $h2-font-size;
+      }
+      
+      .profile-container {
+        position: relative;
+        display: flex;
+        align-items: center;
+
+        .profile-icon {
+          font-size: 1.5rem;
+          cursor: pointer;
+          color: $grey;
+        }
+
+        .logout-button {
+          display: none;
+          position: absolute;
+          top: 100%;
+          right: 0;
+          margin-top: 8px;
+          :deep(.crush-button.crush-primary) {
+            background-color: $white;
+            border: 1px solid $grey;
+          }
+
+          span {
```

Comment:
It is recommended that you use class in this tag cause our vue version 3.4.21 has a conflict with styles and is possible that you are changing things that you are not considering